opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -72,16 +72,23 @@ export const findAll = async (context, user, args) => {
     throw UnsupportedError('Cant find stixCoreObject only based on relationship type, elementId is required');
   }
   let filters = args.filters ?? [];
-  if (isNotEmptyField(args.elementId)) {
+  const elementIds = isNotEmptyField(args.elementId) ? [args.elementId] : [];
```

Comment:
Not sure why we have all those change.
The code in StixDomainObject is much simplier, can you check that either your code is too complex or stixDomainObject doesn't work as expected ?

Reply:
"elementId" is sent in the filters, and it wasn't handled at all, I had to change it and this is one way I found to make it work, did you test it ?

Reply:
I think you missed my point. Either:

- Your code works but its too complex
- The code in src/domain/stixDomainObject doesn't work

Because the goal is the same. Can you check ?

Reply:
updated and it works with StixCoreObjectsFilter, but I didn't add the new param to handle the relationship_type